pkg/external_repos/introspect_test.go

```diff
@@ -25,7 +25,7 @@ func TestIsRedHatUrl(t *testing.T) {
 }
 
 // https://packages.cloud.google.com/yum/repos/google-compute-engine-el8-x86_64-stable/repodata/repomd.xml
-//go:embed "test_files/test_repomd.xml"
+// go:embed "test_files/test_repomd.xml"
```

Comment:
I think this space is evoking the current error into the tests: https://github.com/content-services/content-sources-backend/actions/runs/3543726621/jobs/5950420530#step:5:117

Reply:
Yeah, I noticed this.. Fixed